core/src/test/scala/integration/kafka/api/BaseProducerSendTest.scala

```diff
@@ -227,26 +227,6 @@ abstract class BaseProducerSendTest extends KafkaServerTestHarness {
     }
   }
 
-  @Test
-  def testWrongSerializer() {
```

Comment:
Sure, I had to update this test because of the serializer change in `createNewProducer`. I then realised that I forgot to move this test to `PlaintextProducerSendTest` (like I did with `testSerializerConstructors`) since it is testing the constructor and we don't need to run it for all security protocols.